testsuite/testsuite.c

```diff
@@ -59,7 +59,7 @@ static const char *test_rootfs(const struct test *t)
 	if (t->config[TC_ROOTFS] == NULL)
 		return NULL;
 
-	snprintf(dirname, sizeof(dirname), "%s", t->config[TC_ROOTFS]);
+	snprintf(dirname, sizeof(dirname), TESTSUITE_ROOTFS "%s", t->config[TC_ROOTFS]);
```

Comment:
oh... these functions make absolute no sense before this commit. And still it's very odd to save things in this static variable. Can we just open TESTSUITE_ROOTFS in the beginning and switch everything to use openat()?

Reply:
It's a good idea which crossed my mind.

Plumbing the `fd` across the dozen+ functions will be quite hairy. Alternatively is to use global variable ... of which we don't have any ATM.

Will go with the global var, unless there are other suggestions.